### ISSUE TITLE
Segmentation fault during test only on travis

### ISSUE THREAD

**Issue by tobiasfrust** [tobiasfrust/shibboleth-authenticator#14]
Only on Travis CI build the test fail with a segmentation fault:

```bash
============================= test session starts ==============================
platform linux2 -- Python 2.7.9, pytest-3.0.7, py-1.4.33, pluggy-0.4.0 -- /home/travis/virtualenv/python2.7.9/bin/python
cachedir: .cache
rootdir: /home/travis/build/tobiasfrust/shibboleth-authenticator, inifile: pytest.ini
plugins: pep8-1.0.6, cov-2.5.0
collected 13 items 
tests/conftest.py PASSED
tests/helpers.py PASSED
tests/test_app.py PASSED
tests/test_app.py::test_version PASSED
tests/test_app.py::test_init PASSED
tests/test_utils.py PASSED
tests/test_utils.py::test_accountinfo PASSEDPath: /tmp/tmpvfTZ9s
tests/test_views.py PASSED
tests/test_views.py::test_login PASSEDPath: /tmp/tmpYbsxg0
tests/test_views.py::test_redirect_uri PASSEDPath: /tmp/tmpBT44Ko
tests/test_views.py::test_authorized PASSEDPath: /tmp/tmp7z5NDQ
tests/test_views.py::test_valid_authorized Segmentation fault (core dumped)
The command "sh run-tests.sh" exited with 139.
```
Locally, using drone.io as well as in a manual run all tests pass.

**Comment by tobiasfrust** [comment]
The problem does not occur if travis trusty beta images are used.
```yaml
dist: trusty
```

**Comment by tobiasfrust** [comment]
Resolved in #16.
